docs/explanations/architecture/styles.md

```diff
@@ -483,52 +461,52 @@ In addition to the CSS Custom Properties, all presets but duotone generate CSS c
 .wp-block-site-title .has-foreground-border-color { border-color: var(--wp--preset--color--foreground) !important; }
 ```
 
-#### Current limitations of the Global Styles API
+### Current limitations of the Global Styles API
```

Comment:
```suggestion
### Current limitations of the Global styles API
```

Reply:
Is "Global Styles" a proper noun?

Reply:
Yeah, I think it is because Global Styles refers to a section of the Site Editor as well. Either way, I would consider the "Global Styles API" as a proper noun, much like the "Block Supports API"